tests/raw_query/tests.py

```diff
@@ -413,7 +420,18 @@ def test_bool(self):
         )
 
     def test_len(self):
-        self.assertEqual(len(Book.objects.raw("SELECT * FROM raw_query_book")), 4)
+        self.assertEqual(len(Book.objects.raw("SELECT * FROM raw_query_book")), 5)
         self.assertEqual(
             len(Book.objects.raw("SELECT * FROM raw_query_book WHERE id = 0")), 0
         )
+
+    def test_as_subquery(self):
```

Comment:
As `RawQuery` is not part of the public API, and from what I can see, not tested directly. Instead I think we should have:
```python
    def test_as_subquery(self):
        author_id_sub_query = Book.objects.raw(
            "SELECT id FROM raw_query_author WHERE first_name LIKE %s",
            using="default",
            params=["Bob"],
        )
        with self.assertNumQueries(1):
            self.assertEqual(
                len(Book.objects.filter(author_id__in=author_id_sub_query)), 1
            )
```

I tweaked the query so that we don't need to create a new book object and edit the other tests :+1: